tsconfig.json

```diff
@@ -29,6 +29,7 @@
     "resolveJsonModule": true,
     "esModuleInterop": true
   },
+  "files": ["node_modules/@emotion/react/types/css-prop.d.ts"],
```

Comment:
Include this declaration file in our loaded types. Check out [the Emotion v11 notes about why this is required](https://emotion.sh/docs/emotion-11#types-overhaul).